soc/v2/rockchip/rk3568/soc.yml

```diff
@@ -0,0 +1,4 @@
+series:
```

Comment:
why is each soc having its own soc.yml, instead of the vendor top-level folder having containing a single soc.yml for the whole family and series.

Similar to what we do here:
https://github.com/tejlmand/zephyr/blob/nrf52_adopt_soc_scheme_v2/soc/v2/nordic_nrf/soc.yml

That would also solve the other issue related to not-sourced Kconfig files.

Reply:
Fixed